recipes/openmotif/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "openmotif" %}
+{% set version = "2.3.7" %}
+{% set sha256 = "8f7aadbb0f42df2093d4690735a2b9a02ea2bf69dfb15ae0a39cae28f1580d14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://downloads.sourceforge.net/project/motif/Motif%20{{ version }}%20Source%20Code/motif-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # "Only weak aliases are supported on darwin"
+    # From https://raw.githubusercontent.com/Homebrew/formula-patches/b10858b/openmotif/patch-lib-XmP.h.diff
+    - patch-lib-XmP.h.diff
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - pkg-config
+    - flex
+    - freetype 2.7|2.7.*
+    - fontconfig 2.12.*
+    - libpng >=1.6.28,<1.7
+    - jpeg 9*
+    - xorg-libxt
+    - xorg-libxext
+    - xorg-libxft
+    - xorg-xbitmaps
+    - xorg-libxp
+    - xorg-libxmu
+    - xorg-makedepend
+    - libiconv
+
+  run:
+    - freetype 2.7|2.7.*
+    - fontconfig 2.12.*
+    - libpng >=1.6.28,<1.7
+    - jpeg 9*
+    - xorg-libxt
+    - xorg-libxext
+    - xorg-libxft
+    - xorg-libxp
+    - xorg-libxmu
+    - libiconv
+
+outputs:
+  - name: openmotif-dev
+    script: install_dev.sh
+    requirements:
+      - openmotif
+  - name: openmotif
+    script: install_main.sh
+  - name: openmotif-doc
+    script: install_doc.sh
+    requirements:
+      - openmotif
+
+test:
+  commands:
+    - 'test "`uil 2>&1| tr -d ''\n''`" = "Severe: no source file specified"'
```

Comment:
Running the linter locally can help you figure out the problem but if I had to guess it would be this line.
All thos `, " and ' maybe be messing up the recipe parser.